### PR TITLE
Removed hide personal lists removing EXO license in manage-tasks-app.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5142

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-tasks-app.md [hunk 1]
```diff
@@ -69,9 +69,8 @@ App setup policies let you customize Teams to highlight the apps that are most i
 
 To pin the Tasks app for your users, you can edit the global (Org-wide default) policy or create and assign a custom app setup policy. To learn more, see [Manage app setup policies in Teams](teams-app-setup-policies.md).
 
-### Hide users' Personal lists if you don't want users to see them 
-
-If you don't want users to see Personal lists, you can hide it. To do this, [remove the user's Exchange Online license](https://docs.microsoft.com/microsoft-365/admin/manage/remove-licenses-from-users). Note that after you remove an Exchange Online license, the user no longer has access to their mailbox. Mailbox data is held for 30 days, after which the data will be removed and can't be recovered unless the mailbox is placed on [In-Place Hold or Litigation Hold](https://docs.microsoft.com/exchange/security-and-compliance/in-place-and-litigation-holds).
+> [!NOTE]
+> Personal lists will be visible for any user licensed for Exchange Online, currently there is no supported way to hide them.
```
Comment (JohanFreelancer9) [inline Teams/manage-tasks-app.md:73]:
```suggestion
> Personal lists will be visible for any user licensed for Exchange Online; currently, there is no supported way to hide them.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Followed up with PM.

**Comment by LanaChin** [comment]
Hi @dariomws - I had a conversation with the PM about this and updated the text based on the PM's feedback.  (This is a technically-supported option).

**Pull request merged**